Improve the Content of the Community Page (#131)
ballerina-platform/ballerina-dev-website: community.md

## Want to report issues? 

Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!

> - Found a bug in the compiler, standard library, runtime, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>.

Reviewer: Shall we swap runtime and standard library?
```suggestion
- Found a bug in the compiler, runtime, standard library, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>.
```

Reviewer: IMO, we need "here" and the link should be for that.

Otherwise, we are just repeating "create an issue". We are creating an issue for all the cases, what changes is where.

Author: Normally, it's not a best practice in content writing to give the link saying "here". The guideline is to have a text with a hyperlink.

Reviewer: +1, @pubudu91 suggested something similar.

Also, how about saying  "Create an issue in the ballerina-lang repo" instead?

Author: IMHO, mentioning "the repo" in the hyperlink text will be a bit redundant as the user will click the link anyway if he/she wants to create an issue irrespective of it being a GitHub repo or not.

@pubudu91 - What's your suggestion, please?

Reviewer 2: No, I don't think Maryam meant including "repo" part in the hyperlink. 
Something like below:
"Found a bug in the compiler, standard library, runtime, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repository."

Author: Yes, even saying "repository" without the hyperlink will be redundant for the same reason.  The user will click the link anyway if he/she wants to create an issue irrespective of it being a GitHub repo or not.

Reviewer: ballerina-lang is just the name, reads a bit weird to me.

Author: +1 Changed as follows:

- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo.

- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.

> - Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 

Reviewer 3: IMO we need to include details on how to report security issues here also by linking ballerina security policy.

Author: Added as follows:

`Found a security flaw? Send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.`

## Want to help us make Ballerina popular?!

We know getting a language established is like getting a new religion going. We really need you, our fans, to help spread the word.

> Let’s start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!

Reviewer: ```suggestion
Let's start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!
```

Next, let's share the wealth by publishing your code on <a href="https://central.ballerina.io">Ballerina Central</a>. Hope you will do it now.

> Also, follow us on Twitter:" <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the “#ballerinalang" hashtag.

Reviewer: ```suggestion
Also, follow us on Twitter:" <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.
```

> Also, follow us on Twitter:" <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.

Reviewer: ```suggestion
Also, follow us on Twitter: <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.
```

Oh no, you hit a bump on the road! Please tell us - we want to fix all bugs!

> - Found a bug in the compiler, standard library or the runtime? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">https://github.com/ballerina-platform/ballerina-lang/issues</a>

Reviewer 3: Should we include IDE tooling also here? @kaviththiranga

Reviewer 4: +1

Author: @sanjiva - Shall we add Tooling also here? WDYT?

Author: +1 Added tooling as follows:

"Found a bug in the compiler, standard library, runtime, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>."
